discord/voice_client.py

```diff
@@ -44,21 +44,23 @@
 import logging
 import struct
 import threading
-from typing import Any, Callable, List, Optional, TYPE_CHECKING, Tuple, Union
+import select
+from typing import Any, Callable, List, Optional, TYPE_CHECKING, Tuple, Union, Dict
```

Comment:
Please run `black` and `isort` on the codebase as is the standard.

Reply:
Will do.